src/config.ts

```diff
@@ -109,7 +109,7 @@ export function validateConfig(cfg: Partial<Config>): Config {
         },
         enableTypes: ['form']
       })
-    },
+    } as any,
```

Comment:
Why is this `as any` required now?

Reply:
woops, it's not needed, I've removed this